components/infobox/wikis/starcraft2/infobox_league_custom.lua

```diff
@@ -0,0 +1,590 @@
+---
+-- @Liquipedia
+-- wiki=starcraft2
+-- page=Module:Infobox/League/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local League = require('Module:Infobox/League')
+local String = require('Module:String')
+local Template = require('Module:Template')
+local Variables = require('Module:Variables')
+local Autopatch = require('Module:Automated Patch')._main
+local Tier = require('Module:Tier')
+local Namespace = require('Module:Namespace')
+local AllowedServers = require('Module:Server')
+local RaceIcon = require('Module:RaceIcon')
+local PageLink = require('Module:Page')
+local Class = require('Module:Class')
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+local Title = require('Module:Infobox/Widget/Title')
+local Center = require('Module:Infobox/Widget/Center')
+local Breakdown = require('Module:Infobox/Widget/Breakdown')
+
+local CustomLeague = Class.new()
+local CustomInjector = Class.new(Injector)
+
+local _args
+
+local _ABBR_USD = '<abbr title="United States Dollar">USD</abbr>'
+local _TODAY = os.date('%Y-%m-%d', os.time())
+
+local _GAME_WOL = 'wol'
+local _GAME_HOTS = 'hots'
+local _GAME_LOTV = 'lotv'
+local _GAME_MOD = 'mod'
+
+local _GAMES = {
+	[_GAME_WOL] = {'Wings of Liberty', 'WoL'},
+	[_GAME_HOTS] = {'Heart of the Swarm', 'HotS'},
+	[_GAME_LOTV] = {'Legacy of the Void', 'LotV'},
+	[_GAME_MOD] = {'mod', 'mod'}
+}
+local _SICON = '[[File:Sicon.png|text-bottom|Code S|link=Code S]]'
+local _AICON = '[[File:Aicon.png|text-bottom|Code A]]'
+local _PICON = '[[File:PIcon.png|text-bottom|Premier League]]'
+local _CICON = '[[File:CIcon.png|text-bottom|Challenger League]]'
+
+function CustomLeague.run(frame)
+	local league = League(frame)
+	_args = league.args
+
+	league.getChronologyData = CustomLeague.getChronologyData
+	league.createWidgetInjector = CustomLeague.createWidgetInjector
+	league.defineCustomPageVariables = CustomLeague.defineCustomPageVariables
+	league.addToLpdb = CustomLeague.addToLpdb
+	league.shouldStore = CustomLeague.shouldStore
+
+	return league:createInfobox(frame)
+end
+
+function CustomLeague:createWidgetInjector()
+	return CustomInjector()
+end
+
+function CustomInjector:parse(id, widgets)
+	if id == 'server' then
+		return {
+			Cell{name = 'Game version', content = {
+					CustomLeague._getGameVersion()
+				}},
```

Comment:
This seems strange in a branch for "server"

Reply:
yeah but i (and the other sc2 cntributors) want it moved up there (like in the old infobox), so this would work as intended
might add a comment in the code for it